src/sig_stfl/xmss/external/namespace.h

```diff
@@ -0,0 +1,14 @@
+#ifndef XMSS_NAMESPACE_H
+#define XMSS_NAMESPACE_H
+
+
+#define XMSS_PARAMS oqs_sig_stfl_xmss
+#define XMSS_PARAMS_INNER oqs_sig_stfl_xmss_inner
+
+#define XMSS_NAMESPACE(funcname) XMSS_(XMSS_PARAMS, funcname)
+#define XMSS_INNER_NAMESPACE(funcname) XMSS_(XMSS_PARAMS_INNER, funcname)
+#define XMSS_(prefix, funcname) XMSS__(prefix, funcname)
+#define XMSS__(prefix, funcname) prefix ## _ ## funcname
```

Comment:
If I understand this correctly, this namespaces everything with `oqs_sig_stfl_xmss` or `oqs_sig_stfl_xmss_inner`, but has no variant-specific namespacing.

Reply:
Yup, I intended to share the code between parameters, so there is no plan for parameter namespace. The `_inner` prefix is mainly to indicate the internal functions of `xmss` so it's easy to distinguish them while debugging.